### PR TITLE
chore: minor clarifications in the WP notebook

### PR DESCRIPTION
**Pull request by bcm-at-zama**
Small clarifications in the WP notebook

### CODE REVIEW

File: use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb [hunk 1]
```diff
@@ -577,9 +577,7 @@
    "source": [
     "## Conclusion\n",
     "\n",
-    "Here is a recap of the results obtained by running this notebook, compared to the results in the whitepaper [1].\n",
-    "\n",
-    "A pre-computed comparison to a `hpc7a` instance with 192 vCPU is also shown when running this notebook on a large server machine."
+    "Here is a recap of the results obtained by running this notebook on a `m6i` instance, compared to the results in the whitepaper [1]."
```
Comment (RomanBredehoft) [inline use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb:580]: I would specify `m6i.metal` here as there are several m6i possible instances

File: use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb [hunk 2]
```diff
@@ -639,10 +637,17 @@
...
-    "df, fmt = format_results_df(PAPER_NOTES, results_cml, \"Our\")\n",
+    "df, fmt = format_results_df(PAPER_NOTES, results_cml, \"Our m6i\")\n",
     "df.style.format(fmt)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A pre-computed comparison to a `hpc7a` instance with 192 vCPU is also shown when running this notebook on a large server machine."
```
Comment (RomanBredehoft) [inline use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb:648]: same, I would specify `hpc7a.96xlarge` here

**Review by RomanBredehoft — CHANGES_REQUESTED** [review]
I would be more precise on the machines we are using !

File: use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb (hunk 1 above)
Comment (bcm-at-zama) [reply use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb:580]: done

File: use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb (hunk 2 above)
Comment (bcm-at-zama) [reply use_case_examples/white_paper_experiment/WhitePaperExperiments.ipynb]: done

**Comment by bcm-at-zama** [comment]
Done @RomanBredehoft

**Review by RomanBredehoft — APPROVED** [review]
thanks !

**Comment by bcm-at-zama** [comment]
Can we merge that please @andrei-stoian-zama

**Pull request merged**